src/scripts/publish.sh

```diff
@@ -21,7 +21,7 @@ function validateOrbPubToken() {
 
 function publishOrb() {
   #$1 = full tag
-  circleci orb publish --skip-update-check "${ORB_PARAM_ORB_DIR}/orb.yml" "${ORB_PARAM_ORB_NAME}@${1}" --token "$ORB_PARAM_ORB_PUB_TOKEN"
+  circleci orb publish --host "${CIRCLECI_API_HOST}" --skip-update-check "${ORB_PARAM_ORB_DIR}/orb.yml" "${ORB_PARAM_ORB_NAME}@${1}" --token "$ORB_PARAM_ORB_PUB_TOKEN"
```

Comment:
Sets default to circleci.com

```suggestion
  circleci orb publish --host "${CIRCLECI_API_HOST:-https://circleci.com} --skip-update-check "${ORB_PARAM_ORB_DIR}/orb.yml" "${ORB_PARAM_ORB_NAME}@${1}" --token "$ORB_PARAM_ORB_PUB_TOKEN"
```